docs/data/base/guides/working-with-tailwind-css/working-with-tailwind-css.md

```diff
@@ -305,7 +310,7 @@ Let's add the `Slider` into the `Player` component now:
 
 You should see this:
 
-<img src="/static/images/base-with-tailwind-css/player-slider.png" alt="Screenshot of the media player used as example in the guide, designed by the Tailwind Labs team" style="width: 796px; margin-top: 8px; margin-bottom: 8px;" />
+{{"demo": "PlayerInteractive.js", "hideToolbar": true, "bg": true}}
```

Comment:
What do you think about having a screenshot instead of a codesandbox? My concern is that it might become a pain to keep 3 codesandboxes up to date. At the same time, maybe having one codesandbox instead of zero might already yield most of the value?

Reply:
I also have doubt about having more codesandboxes, but have created them for the first iteration. Will revert back the images for the rest.